platform/features/table/bundle.js

```diff
@@ -0,0 +1,132 @@
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define([
+    "./src/directives/MCTTable",
+    "./src/controllers/TableController",
+    "./src/controllers/TableOptionsController",
+    "./src/controllers/MCTTableController",
+    '../../commonUI/regions/src/Region',
+    '../../commonUI/browse/src/InspectorRegion',
+    "legacyRegistry"
+], function (
+    MCTTable,
+    TableController,
+    TableOptionsController,
+    MCTTableController,
+    Region,
+    InspectorRegion,
+    legacyRegistry
+) {
+    "use strict";
+    /**
+     * Two region parts are defined here. One that appears only in browse
+     * mode, and one that appears only in edit mode. For not they both point
+     * to the same representation, but a different key could be used here to
+     * include a customized representation for edit mode.
+     */
+    var tableInspector = new InspectorRegion(),
+        tableOptionsEditRegion = new Region({
+            name: "table-options",
+            title: "Table Options",
+            modes: ['edit'],
+            content: {
+                key: "table-options-edit"
+            }
+        });
+    tableInspector.addRegion(tableOptionsEditRegion);
+
+    legacyRegistry.register("platform/features/table", {
+        "extensions": {
+            "types": [
+                {
+                    "key": "table",
+                    "name": "Table",
+                    "glyph": "\ue605",
+                    "description": "A table for displaying telemetry data",
+                    "features": "creation",
+                    "delegates": [
+                        "telemetry"
+                    ],
+                    "inspector": tableInspector,
+                    "contains": [
+                        {
+                            "has": "telemetry"
+                        }
+                    ],
+                    "model": {
+                        "composition": []
+                    },
+                    "views": [
+                        "table"
+                    ]
+                }
+            ],
+            "controllers": [
+                {
+                    "key": "TableController",
+                    "implementation": TableController,
+                    "depends": ["$scope", "telemetryHandler", "telemetryFormatter"]
+                },
+                {
+                    "key": "TableOptionsController",
+                    "implementation": TableOptionsController,
+                    "depends": ["$scope"]
+                },
+                {
+                    "key": "MCTTableController",
```

Comment:
One problem to registering this controller is it takes up namespace and is not reusable outside of the directive.  Would recommend loading this in the directive via require and specifying dependencies with the `$inject` property.  Keep the TableDirective as a single component that only exposes a single interface.